cms/djangoapps/contentstore/views/tests/test_videos.py

```diff
@@ -364,21 +362,14 @@ def _check_csv_response(self, expected_video_ids, expected_profiles):
                     self.assertEqual(response_profile_url, original_encoded_for_profile["url"])
                 else:
                     self.assertEqual(response_profile_url, "")
-        self.assertEqual(set(actual_video_ids), set(expected_video_ids))
+        self.assertEqual(len(actual_video_ids), len(self.previous_uploads))
```

Comment:
This could continue to use set comparisons, but that would require more code to extract the set of video ids from self.previous_uploads. The assertions above ensure that each row contains a unique video id that occurs in self.previous_uploads, so this condition is sufficient to determine that the sets are equal.